model/scikit/dffml_model_scikit/scikit_base.py

```diff
@@ -119,6 +121,94 @@ def _filename(self):
             yield repo
 
 
+class ScikitContextUnsprvised(ScikitContext):
+    def __init__(self, parent):
+        super().__init__(parent)
+
+    def _feature_predict_hash(self):
+        params = "".join(
+            [
+                "{}{}".format(k, v)
+                for k, v in self.parent.config._asdict().items()
+                if k not in ["directory", "features"]
+            ]
+        )
+        return hashlib.sha384(
```

Comment:
qq, why is a crypto hash used here? , wouldn't a regular md5 or a faster hash suffice?

Reply:
Ah! Everywhere in the `model` `sha384` has been used. I wrote it without even realizing that it might be a bit of overkill. But honestly, I am not sure. I guess md5 may suffice but using `sha384` here almost guarantees no collision.

Reply:
yeah sha384 is an overkill to just to prevent collisions, also collisions here cannot be considered a show stopper or `threat, I would suggest using something like SHA1 if you have concerns with md5 and if you need added locality salt and hash. @pdxjohnny could comment better, there might be reasons for using SHA384 that I am not aware of.

Reply:
(This was the comment I meant to post, oops sorry) We do `sha384` this almost everywhere to avoid getting hit by bandit scans. Let's move to doing https://github.com/intel/dffml/issues/297 longterm